src/main/main.ts

```diff
@@ -77,13 +77,31 @@ const handleError = (
   errorCode: ErrorCode
 ) => {
   const reply: TError = { errorCode };
+  console.log(channel);
```

Comment:
extra console log?

Reply:
I want to leave this in somehow. If someone calls handleError, especially with the Failure errorcode, we have no idea where it came from.